src/main/resources/application.yml

```diff
@@ -43,6 +43,8 @@ app:
     siteKey: 6LetqBAgAAAAAJXA5K_U88bsxKtyp_vld6J0x-Nv
     apiKey: secret
     magicToken: secret
+  chatDancer:
```

Comment:
here we need to distuinguish between local, staging and live. We could have an .env file for every environment...

Reply:
No, the chat-dancer will we not accessible at all from the internet (remember ;-) ) So it will also we resolvable as chat-dancer ;-)

Reply:
Oh, I am making so many typos :-( Shame on me.